commands/channel/farewell.py

```
@@ -16,10 +12,17 @@
     set_leave_channel,
 )
 from localizer import tanjunLocalizer
+from services.pillow_service import (
```

Flake8 (7.3.0)

[error] 'services.pillow_service.fetch_image' imported but unused

(F401)